src/Throwable/Exception/FixtureNotFoundException.php

```diff
@@ -13,6 +13,8 @@
 
 namespace Nelmio\Alice\Throwable\Exception;
 
-class FixtureNotFoundException extends \UnexpectedValueException
+use Nelmio\Alice\Throwable\ResolutionThrowable;
+
+class FixtureNotFoundException extends \UnexpectedValueException implements ResolutionThrowable
```

Comment:
I would rather create a new exception for the resolver context: it's doesn't make sense otherwise